src/tests/common.py

```diff
@@ -13,7 +13,7 @@
 def cleanup(home=None, files=_files):
     """Cleanup application files"""
     if not home:
-        import state
+        from pybitmessage import state
```

Comment:
you've already got the pathmagic for this: the `common` module is used in both test suites
 - in the external tests we write `from pybitmessage import ..`
 - but it the core tests it's still `import state`

I suppose we gonna switch to `from . import state` inside of the `pybitmessage` package and using another start script in top level dir, but not earlier than the circular import in the `network` package is solved.